docs/designers-developers/developers/backward-compatibility/README.md

```diff
@@ -1 +1,57 @@
+
+### How to guarantee backward compatibility for a React Component
+
+* The name of the component should not change.
+* The props of the components should not be removed.
+* Existing prop values should continue to be supported. If a component accept a function as prop, we can update the component to accept a new type for the same prop, but it shouldn't break existing usage.
+* Adding new props is allowed.
+* React Context dependencies can only be added or removed if we ensure the previous context contract is not breaking.
+* Classnames, DOM nodes used inside the tree of the component are not considered part of the public API and can be modified. That said, changes to these should be done with caution as it can affect the styling. When classes need to be renamed, if possible keep the old ones. When possible, document these changes and write a devnote about the changes.
+
+### How to guarantee backward compatibility for a Block
+
+* Existing usage of the block should not break or be marked as invalid when the editor is loaded.
+* The styling of the existing blocks should be guaranteed.
+* Markup changes should be limited to the minimum but If a block need to change its saved markup making previous versions invalid, a **deprecated version** for the block should be added.
+
+## Deprecations
+
+As the project evolves, flaws to existing APIs are discovered, or updates are required to support new features. When this happens, we try to guarantee that existing APIs don't break and build new alternative APIs.
+
+To encourage third-party developpers to adopt the new APIs instead, we can use the **deprecated** helper to show a message explaining the deprecation and proposing the alternativewhenever the old API is used.
+
+## Dev Notes
+
+Dev Notes are posts published in the make/core site prior to WordPress releases to inform third-party developpers about important changes to the developper APIs, these changes can include:
+* Promote new APIs.
+* Explain how some changes to existing APIs might affect existing plugins and themes (Example: classname changes...)
+* When breaking backward compatibility is unavoidable, explain the reasons and the migration flow.
+* When important deprecations are introduced (even without breakage), explain why and how third-party developpers can update their code base in consequence.
+
+### Dev Note Workflow
+
+- When working on Pull-request and the need for a dev-note is noted/discovered, add the **Needs Dev Note** label to the PR.
+- If possible, add a comment to the PR explaining why the Dev Note is needed.
+- When the first beta of the next WordPress release is shipped, go throught the list of merged PRs included in this release and tagged with the **Needs Dev Note** label.
+- For each one of these PRs, write a Dev Note, coordinate with the WordPress release leads to publish the dev note.
```

Comment:
```suggestion
- For each one of these PRs, write a dev note, and coordinate with the WordPress release leads to publish the dev note.
```

Reply:
I think before `and` there shouldn't be a comma?